ipa-core/tests/common/mod.rs

```diff
@@ -261,14 +267,27 @@ pub fn test_ipa_with_config(mode: IpaSecurityModel, https: bool, config: IpaQuer
         .args([
             "--per-user-credit-cap",
             &config.per_user_credit_cap.to_string(),
-        ])
-        .stdin(Stdio::piped());
+        ]);
+
+    match config.with_dp {
+        0 => {
+            command.args(["--with-dp", &0_u32.to_string()]);
+        }
+        _ => {
+            command
+                .args(["--with-dp", &1_u32.to_string()])
```

Comment:
is there a reason for hardcoding `1` here or we could pass `with_dp` value instead?

Reply:
yes, we should be able to do that. Will update